package.json

```diff
@@ -80,6 +80,7 @@
     "raf-schd": "^4.0.3",
     "react-redux": "^8.1.1",
     "redux": "^4.2.1",
+    "ts-node": "^10.9.1",
```

Comment:
oh I didn't notice that I committed this, but it was required to run `yarn test`